### PR TITLE
T10 - Do Important Things

### PR DESCRIPTION
**Pull request by starlingjon**
This comment describes the ticket T10 and what was done to fix the problem/ accomplish the task/ etc. It could very well be copied from zoho, though it probably makes more sense to keep it short and concise as we have put the ticket number in the title (and probably given the PR the same title as the ticket itself)

### CODE REVIEW

File: someOtherFile.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+this is another file
+
+being added for ticket t10
+
+on the t10 branch
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
```
Comment (jonmackenzie) [inline someOtherFile.txt:10]: This is a comment asking a question, like "why not do X?" or "are you sure this will work?"

File: somefile.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+created for ticket t10
+
+on the t10 branch
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+nisi ut aliquip ex ea commodo consequat.
+
+this line added in a second commit to the t10 branch, before CR
```
Comment (jonmackenzie) [inline somefile.txt:14]: This is a comment about this suggestion
```suggestion
this line was added in a second commit to the t10 branch, before CR
```

**Review by jonmackenzie — CHANGES_REQUESTED** [review]
This is a comment with general feedback

File: someOtherFile.txt (hunk 1 above)
Comment (starlingjon) [reply someOtherFile.txt:10]: How about this?
```suggestion
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
```

Comment (starlingjon) [reply someOtherFile.txt:10]: How about this?

```suggestion
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
Tempor incididunt ut labore et dolore magna aliqua.
```

Comment (jonmackenzie) [reply someOtherFile.txt:10]: Looks good to me

**Review by jonmackenzie — APPROVED** [review]
Looks good to me

**Pull request merged**